commit cc73b0e4a8f4b90f63289897f6f1102573108087
Author: John Doe <jdoe@Johns-Mac.local>
Date:   Sun Feb 5 20:47:31 2017 -0500

    added AM_PROG_AR and AC_CONFIG_MACRO_DIR([m4])  to configure.ac

diff --git a/ChangeLog b/ChangeLog
new file mode 100644
index 0000000..e69de29
diff --git a/configure.ac b/configure.ac
index 02158de..e1ce98c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -22,8 +22,12 @@ AC_DEFINE(PYCAIRO_VERSION_MICRO, pycairo_version_micro, [pycairo macro version])
 AC_CONFIG_SRCDIR([src/pycairo.h])
 AC_CONFIG_HEADERS(src/config.h)
 
+AC_CONFIG_MACRO_DIR([m4])
+
 AM_INIT_AUTOMAKE([1.9.6 -Wall])
 
+AM_PROG_AR
+
 # Options ---
 
 # put the ACLOCAL flags in the makefile
